Formula/git.rb

```diff
@@ -4,6 +4,7 @@ class Git < Formula
   url "https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.37.2.tar.xz"
   sha256 "1c3d9c821c4538e7a6dac30a4af8bd8dcfe4f651f95474c526b52f83406db003"
   license "GPL-2.0-only"
+  revision 1
```

Comment:
If it's a build dependency, do we need the `revision` bump?

Reply:
We don't need one anymore - I forgot to remove this.